cubed/backend_array_api.py

```diff
@@ -0,0 +1,23 @@
+import numpy as np
```

Comment:
I'm excited to see this as simple place to drop in a Jax array implementation :) Great work.

Reply:
It will be interesting to see if this works with the [recent work in JAX on the Array API](https://github.com/google/jax/pull/16099).